### PR TITLE
libbulletc: fix method signatures, missing include, brace on wrong li…

### PR DESCRIPTION
**Pull request by Alex-----**
…ne, add double precision option to CMake and missing files to library.

Fixes some bugs I found. I also commented out all the #define BULLETC_DISABLE at the top of main.h, not included in commit. Also added the double precision option to CMake, I don't know if you want that. It compiles on Windows but I haven't tested the changes. 
There were some other problems that I didn't fix, I think it was some methods that were commented out in the cpp file (like btDbvt_rayTest) that are still in the header file.

### CODE REVIEW

File: libbulletc/CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ ELSE()
     SET(BULLETC_LIB libbulletc)
 ENDIF()
 
+OPTION(USE_DOUBLE_PRECISION "" ON)
```
Comment (AndresTraks) [inline libbulletc/CMakeLists.txt]: In this branch, the .NET part of the library uses single precision, so this option should be off. There is a separate branch for [double precision](https://github.com/AndresTraks/BulletSharpPInvoke/tree/double-precision).

I haven't found an easy way to make the C# project support both float and double in the same code.
Single precision is still suitable for most scenarios, so I'll keep that as the default, like it is in Bullet.

File: libbulletc/CMakeLists.txt [hunk 2]
```diff
@@ -375,6 +376,8 @@ ADD_LIBRARY(${BULLETC_LIB} SHARED
     src/btTriangleIndexVertexArray_wrap.h
     src/btTriangleIndexVertexMaterialArray_wrap.cpp
     src/btTriangleIndexVertexMaterialArray_wrap.h
+    src/btTriangleInfoMap_wrap.cpp
+    src/btTriangleInfoMap_wrap.h
```
Comment (AndresTraks) [inline libbulletc/CMakeLists.txt:379]: These are rarely used outside of Bullet internals and the rest of the wrapper doesn't relate to these classes yet.
But they are exposed publicly in Bullet, so it's OK to include the files.

**Comment by AndresTraks** [comment]
There is some code that is commented out that was automatically generated using a custom code generator, but needs some tweaking to get it to work.
It is mostly methods that are used internally (like the btDbvt stuff), so I'll probably only work on that code if there is a specific need.

File: libbulletc/CMakeLists.txt (hunk 1 above)
Comment (AndresTraks) [reply libbulletc/CMakeLists.txt]: The rest looks good, so if you remove this line, I'll merge the request.
Thanks!

Comment (Alex-----) [reply libbulletc/CMakeLists.txt]: I removed it.

**Pull request merged**